pipe-cli/mount/pipefuse/pipefs.py

```diff
@@ -150,12 +176,14 @@ def statfs(self, path):
             'f_namemax': 255
         }
 
+    @syncronized
     def unlink(self, path):
         self.client.delete(path)
 
     def symlink(self, name, target):
         raise UnsupportedOperationException("symlink")
 
+    @syncronized
     def rename(self, old, new):
```

Comment:
During the `rename` operation we lock only on the old path. Most likely it is harmless but probably we can lock both paths. But... it can cause deadlocks if two processes rename same files. What do you think?